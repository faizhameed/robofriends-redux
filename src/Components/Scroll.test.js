import { shallow } from "enzyme";
import React from "react";
import Scroll from "./Scroll";
import toJson from "enzyme-to-json";

it("expect to run scroll componen", () => {
  expect(shallow(<Scroll />).length).toEqual(1);
});

it("expect to run Scroll componen", () => {
  const wrapper = shallow(<Scroll />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
