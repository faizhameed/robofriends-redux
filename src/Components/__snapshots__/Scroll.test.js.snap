// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to run Scroll componen 1`] = `
<div
  style={
    Object {
      "border": "0px solid black",
      "height": "500px",
      "overflowY": "scroll",
    }
  }
/>
`;
